src/map/mob.c

```diff
@@ -4719,6 +4712,85 @@ static void mob_drop_ratio_adjust(void){
 }
 
 /**
+ * Copy skill from DB to monster
+ * @param mob Monster DB entry
+ * @param skill Monster skill entries
+ **/
+static void mob_skill_db_set_single_sub(struct mob_db *mob, struct s_mob_skill *skill) {
+	uint8 i;
+
+	nullpo_retv(mob);
+	nullpo_retv(skill);
+
+	for (i = 0; mob->maxskill < MAX_MOBSKILL && i < skill->count; i++) {
+		mob->skill[mob->maxskill++] = skill->skill[i];
+	}
+
+	if (i < skill->count)
```

Comment:
how if u try to import something? somehow my mood is bad today. :P

currently MAX_MOBSKILL is 50, only 2 mobs officially have 50 skills. then try u import some skills for those 2 mobs (up to u the count) or import skills for -1/-2/-3
try debugging it.